REA/Views/GenerateReportsPage.xaml

```diff
@@ -2,37 +2,54 @@
-                        <TextCell Text="Monday - Weather Average Data"
-                      Detail="Speed:6MPH | Wind Direction:78 | Temp:12C | Humidity:98" />
-                        <TextCell Text="Tuesday - Weather Average Data"
-                      Detail="Speed:12MPH | Wind Direction:140 | Temp: 06C | Humidity:78" />
-                        <TextCell Text="Wednesday - Weather Average Data"
-                      Detail="Speed:1MPH | Wind Direction:120 | Temp:18C | Humidity:80" />
-                        <TextCell Text="Thursday -  Weather Average Data"
-                      Detail="Speed:3MPH | Wind Direction:20 | Temp:15C | Humidity:88" /> 
-                        <TextCell Text="Friday - Weather Average Data"
-                      Detail="Speed:3MPH | Wind Direction:190 | Temp:13C | Humidity:99" />
-                    </TableSection>
-                </TableRoot>
-            </TableView>
+                <Label FontSize="18" HorizontalOptions="Center" Text="Water Measurements" Margin="5" TextColor="Blue"></Label>
+                <Border Padding="10" Margin="5">
+                    <VerticalStackLayout>
+                        <Label Text="{Binding Nitrate1, StringFormat='Nitrate: {0}'}" FontSize="16" />
+                        <Label Text="{Binding Nitrate2, StringFormat='Nitrate: {0}'}" FontSize="16"/>
+                        <Label Text="{Binding Phosphate, StringFormat='Phosphate: {0}'}" FontSize="16"/>
+                        <Label Text="{Binding Ec, StringFormat='EC: {0}'}" FontSize="16"/>
+                    </VerticalStackLayout>
+                </Border>
 
-            <!-- This button when clicked, it reads the database again and outputs new data if the database has modifed data-->
-            <Button BackgroundColor="Green" Margin="0,25,0,0" Text="Update Reports (Dummy)"></Button>
-        </StackLayout>
-    </VerticalStackLayout>
+                <Label FontSize="18" HorizontalOptions="Center" Text="Weather Measurements" Margin="5" TextColor="Blue"></Label>
+                <Border Padding="10" Margin="5">
+                    <VerticalStackLayout>
+                        <Label Text="{Binding Temperature2m, StringFormat='Temperature: {0}'}" FontSize="16" />
+                        <Label Text="{Binding RelativeHumidity2m, StringFormat='Relative Humidity: {0}'}" FontSize="16"/>
+                        <Label Text="{Binding WindSpeed10m, StringFormat='Wind Speed: {0}'}" FontSize="16"/>
+                        <Label Text="{Binding WindDirection10m, StringFormat='Wind Direction: {0}'}" FontSize="16"/>
+                    </VerticalStackLayout>
+                </Border>
+            </StackLayout>
+            <StackLayout Margin="15">
+                <Button Text="Calculate Mean" Margin="15" ></Button>
+                <Button Text="Calculate Mode" Margin="15" IsEnabled="False"></Button>
+                <Button Text="Calculate Median" Margin="15" IsEnabled="False"></Button>
```

Comment:
I believe this should be flipped, to indicate that it is the mean that is selected it should be disabled rather than the others. I was confused what this meant initially and thought the mean button does something different. Just a quick UX fix